src/main/java/com/gregtechceu/gtceu/common/data/mui/GTMuiPanels.java

```diff
@@ -17,4 +21,38 @@ public class GTMuiPanels {
         panel.child(new TextWidget(new StringKey("test").scale(10.0f).color(0xff0000)));
         return panel;
     };
+
+    public static PanelFactory BASE_PANEL = (PosGuiData data, PanelSyncManager syncManager, UISettings settings,
```

Comment:
This should be called something related to singleblocks or whatever, since our generic "base_panel" shouldn't have 3 columns with input progress output